ui/lib/core/addon/components/form-field.js

```diff
@@ -153,6 +158,10 @@ export default class FormFieldComponent extends Component {
     this.setAndBroadcast(`${valueToSet}`);
   }
   @action
+  setAndBroadcastDatetimeLocal(evt) {
```

Comment:
instead of adding a new action I think you can just use `onChangeWithEvent` ? which would call `    this.setAndBroadcast(event.target['value'])` since the type is not `checkbox`

Reply:
🤯 thanks for pointing that out Claire! TIL more about form fields :)